docs/devops-guide/log-analyser.md

```diff
@@ -0,0 +1,192 @@
+---
+id: devops-guide-log-analyser
+title: Self-Hosting Guide - Log Analyser
+sidebar_label: Log Analyser
+---
+
+Welcome to the Jitsi Meet Log Analyser project! This integration leverages Grafana Loki and OpenTelemetry to enable effective log management and analysis for Jitsi Meet components.
+
+## Overview
+
+This project offers a streamlined setup for collecting, parsing, and visualizing log data generated by Jitsi Meet components. The integration includes:
+
+- A Docker Compose setup file (`log-analyser.yml`) for configuring Loki and OpenTelemetry Collector.
+- A Docker Compose setup file (`grafana.yml`) for configuring Grafana.
+- A unified Docker Compose command that initiates all services concurrently.
+- Instructions on configuring Grafana with Loki as a log data source for enhanced visualization and analysis.
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, ensure you have the following installed on your system:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Setup
+
+To set up the project, follow these steps:
+
+1. **Clone the repository:**
+
+   Clone the Jitsi Meet repository that contains the necessary Docker Compose files.
+
+    ```bash
+    git clone https://github.com/jitsi/docker-jitsi-meet.git
+    cd docker-jitsi-meet
+    ```
+2. **Update Jitsi Meet Docker Compose Configuration:**
+
+   To enable log collection and analysis, you need to modify the `docker-compose.yml` file for Jitsi Meet components. Add the following configuration to each Jitsi service within the `docker-compose.yml` file:
+
+    ```yaml
+    logging:
```

Comment:
why not use an environment/pass parameter to enable json logging?

Reply:
we decided to add this to Readme for now, but we may fix this part in the future